### ISSUE TITLE
Remove "Import Key" button, make attached keys downloadable

### ISSUE THREAD

**Issue by bnvk** [mailpile/Mailpile#647]
There is a UI button that now shows up if a PGP key is attached to an email as per #623 the next step is to make this properly complete the action once #625 is complete

Currently it pulls up a modal that ask for confirmation. Ideally this modal should show a fingerprint of the attached key

**Comment by BjarniRunar** [comment]
Actually, on second thought, I think this is a misfeature. The button should just go away. Renaming issue to track just that. :-)

Rationale:

1. Users don't care about keys. They shouldn't have to care about keys.
2. Users may care about verifying signatures, issue #1043 would address that.
3. Users may care about encrypting; detecting the key and importing from e-mail is already a working part of the encryption flow.

Basically, if 2. and 3. work, there's no need to manually import the key. The user may need to be able to download the key to use it with some external workflow (so we shouldn't completely suppress the attachment), but we want to spare users the pain of interacting with or worrying about keys if at all possible.
